R/gh_fill.R

```diff
@@ -0,0 +1,14 @@
+gh_fill <- function(geohashes, precision) {
```

Comment:
Please also consider some edge cases:

`geohashes = character()`
`geohashes = NA_character_`
`geohashes = c("0", NA_character_)`
`gh_fill("0", precision = 1L)`

What should be the behavior here?